pre-commit.sh

```diff
@@ -34,10 +34,19 @@ else
     changed_files=`git diff --name-only "${base_commit}..HEAD"`
     dirty_files=`git ls-files -m`
     files_to_check="$((echo "$changed_files"; echo "$dirty_files") | grep '\.py$')"
```

Comment:
please do tiny upgrade - filter out repeated occurences of the same files, that coming from `$changed_files` and also from `$dirty_files`.
replace this line with:
```bash
files_to_check="$((echo "$changed_files"; echo "$dirty_files") | grep '\.py$' | sort | uniq)"
```